README.md

```diff
@@ -305,6 +305,9 @@ The configuration options for the Kafka Connect source connector for IBM MQ are
 | mq.message.mqmd.read                    | Whether to enable reading of all MQMD fields                                                                            | boolean | false          |                                                                                                                                                                                     |
 | mq.max.poll.blocked.time.ms             | How long the connector will wait for the previous batch of messages to be delivered to Kafka before starting a new poll | integer | 2000           | It is important that this is less than the time defined for `task.shutdown.graceful.timeout.ms` as that is how long connect will wait for the task to perform lifecycle operations. |
 | mq.client.reconnect.options             | Options governing MQ reconnection.                                                                                      | string  | ASDEF          | ASDEF, ANY, QMGR, DISABLED                                                                                                                                                          |
+| mq.jms.receive.timeout                  | The timeout in milliseconds for receiving messages from JMS Consumer.                                                   | long    | 2000L          | 1 or greater                                                                                                                                                                        |
```

Comment:
```suggestion
| mq.jms.receive.timeout                  | How long the connector should wait (in milliseconds) for a message to arrive if no message is available immediately                         | long    | 2000          | 1 or greater                                                                                                                                                                        |
```

1. Modified the description by copying from the MQ docs about the wait parameter
2. I think there is potential to confuse someone that they need to add the "L" if we include this